server/views/components/page-description/index.njk

```diff
@@ -2,7 +2,7 @@
   <div class="row row--no-padding {{ 'row--lead row--lead--l page-description__lead' if lead }}">
     <div class="container">
       <div class="grid">
-        <div class="grid__cell {{ 'grid__cell--shift-l1' if lead }}">
+        <div class="grid__cell grid__cell--s4 grid__cell--m6 grid__cell--shift-m1 {{ 'grid__cell--shift-l1 grid__cell--l11 grid__cell--xl10' if lead else 'grid__cell--l12 grid__cell--xl11' }}">
```

Comment:
It'd be nice to come up with a `className` function that deals with this as this is getting a little unwieldy.